tests/issues/issue-1653.php

```diff
@@ -0,0 +1,921 @@
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'ActiveXControls' => false,
+            'isMobileDevice' => false,
+            'isTablet' => false,
+            'isSyndicationReader' => false,
+            'Crawler' => true,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '0',
+            'AolVersion' => '0',
+            'Device_Name' => 'unknown',
+            'Device_Maker' => 'unknown',
+            'Device_Type' => 'unknown',
+            'Device_Pointing_Method' => 'unknown',
+            'Device_Code_Name' => 'unknown',
+            'Device_Brand_Name' => 'unknown',
+            'RenderingEngine_Name' => 'unknown',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'unknown'
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true
+    ],
+    'issue-1653-L' => [
+        'ua' => 'python-requests/2.11.1',
+        'properties' => [
+            'Comment' => 'python-requests',
+            'Browser' => 'python-requests',
+            'Browser_Type' => 'Bot/Crawler',
+            'Browser_Bits' => '0',
+            'Browser_Maker' => 'Python Software Foundation',
+            'Browser_Modus' => 'unknown',
+            'Version' => '2.11',
+            'MajorVer' => '2',
+            'MinorVer' => '11',
+            'Platform' => 'unknown',
+            'Platform_Version' => 'unknown',
+            'Platform_Description' => 'unknown',
+            'Platform_Bits' => '0',
+            'Platform_Maker' => 'unknown',
+            'Alpha' => false,
+            'Beta' => false,
+            'Win16' => false,
+            'Win32' => false,
+            'Win64' => false,
+            'Frames' => false,
+            'IFrames' => false,
+            'Tables' => false,
+            'Cookies' => false,
+            'BackgroundSounds' => false,
+            'JavaScript' => false,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'ActiveXControls' => false,
+            'isMobileDevice' => false,
+            'isTablet' => false,
+            'isSyndicationReader' => false,
+            'Crawler' => true,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '0',
+            'AolVersion' => '0',
+            'Device_Name' => 'unknown',
+            'Device_Maker' => 'unknown',
+            'Device_Type' => 'unknown',
+            'Device_Pointing_Method' => 'unknown',
+            'Device_Code_Name' => 'unknown',
+            'Device_Brand_Name' => 'unknown',
+            'RenderingEngine_Name' => 'unknown',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'unknown'
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true
+    ],
+    'issue-1653-M' => [
```

Comment:
Please remove the duplicate test. This UA is already available on test `issue-1597-NA`